src/index.html

```diff
@@ -2,15 +2,27 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form data-qa="big">
+      <img class="icon" src="images/Search.svg"
+      alt="search" data-qa="keypress">
```

Comment:
Need to fix it everywhere
<img width="1070" alt="image" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/64750958/2a78f725-7612-403a-b852-f9197e2e7ebb">